<!DOCTYPE html>
<!--[if lt IE 7]> <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
<!--[if IE 7]> <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
<!--[if IE 8]> <html class="no-js lt-ie9"> <![endif]-->
<!--[if gt IE 8]> <!-->
<html lang="zxx">
<!--
<![endif]-->

<head>
    <!-- TITLE OF SITE -->
    <title>Gu's Lab</title>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="author" content="cyc">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">


    <!-- CSS Begins
================================================== -->
    <!--Animate Effect-->
    <link href="static/css/animate.css" rel="stylesheet">
    <link href="static/css/hover.css" rel="stylesheet">

    <!-- For Image Preview -->
    <link rel="stylesheet" href="static/css/magnific-popup.css">

    <!--Owl Carousel -->
    <link href="static/css/owl.carousel.css" rel="stylesheet">

    <link rel="stylesheet" href="static/css/slick.css">

    <!--BootStrap -->
    <link href="static/css/bootstrap.min.css" rel="stylesheet">
    <link href="static/css/normalize.css" rel="stylesheet">

    <!-- Main Style -->
    <link href="static/css/style.css" rel="stylesheet">
    <link href="static/css/responsive.css" rel="stylesheet">


    <!--[if IE]>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->

</head>

<body>

    <!-- ::::::::::::::::::::::::::: Start: Preloader section ::::::::::::::::::::::::::: -->
    <div id="preloader"></div>
    <!-- ::::::::::::::::::::::::::: End: Preloader section ::::::::::::::::::::::::::: -->

    <!-- Start: Header Section
==================================================-->


    <!-- Start: header navigation -->
    <div class="navigation">
        <div class="container">
            <div class="logo" style="width: 400px;">
                <a href="index.html">
                    <img class="img-responsive" src="static/picture/logo.png" alt="" style="width: 400px;" >
                </a>
            </div>
            <div id="navigation" class="menu-wrap">
                <ul>
                    <li><a href="index.html">Home</a></li>
                    <li class="active"><a href="">Research Achievement</a></li>
                    <li><a href="Publications.html">Publications</a></li>
                    <li><a href="Software.html"> Software</a></li>
                </ul>
            </div>
            <!-- End: Menu  -->
        </div>
        <!--/ container -->
    </div>
    <!-- End: header navigation -->


<!--   End: Work Flow Section
==================================================-->
    
    <div class="single_course" style="padding-top: 70px;">
        <div class="container">
            <div class="row" style="justify-content: center;">
                <!-- Blog Area -->
                <div class="base-header text-center" style="padding-bottom:20px">
                    <h3 style="font-weight:1000px ;font-size:50px"> Achievement</h3>
                </div>
                <div class="col-lg-10 col-sm-12" >
                    <div class="sing_course_wrap">
                    
                        <div class="course_tab">
                            <div class="course_tab_ul">
                            <ul class="nav nav-pills " id="pills-tab" role="tablist">
                                <li class="nav-item">
                                    <button class="nav-link active" id="pills-discription-tab" data-bs-toggle="tab" data-bs-target="#pills-discription" role="tab" aria-controls="pills-discription" aria-selected="true" type="button">Research Achievement</button>
                                </li>
                                <li class="nav-item">
                                    <button class="nav-link" id="pills-curriculum-tab" data-bs-toggle="tab" data-bs-target="#pills-curriculum" role="tab" aria-controls="pills-curriculum" aria-selected="false" type="button">Highly-Cited Papers</button>
                                </li>
                            </ul></div>
                            <div class="tab-content course_tab_cont" id="pills-tabContent">
                                <div class="tab-pane fade show active" id="pills-discription" aria-labelledby="pills-discription-tab" role="tabpanel">
                                    <h5> Predicting functional divergence between duplicate genes: </h5>
                                    <p style="margin-bottom: 10px;">
                                        In the past 15 years, my research group has developed a series of statistical models 
                                        and algorithms to analyze the pattern of functional divergence between duplicate genes, 
                                        based on the phylogeny-based analysis of multiple protein sequence alignment. 
                                        The underlying premise is that functional divergence may lead to site-by-site 
                                        differences in evolutionary rates between duplicates. A simple case is, for instance, 
                                        that an amino acid site is highly conserved in one duplicate cluster whereas highly variable in the other one. 
                                        We integrated these molecular evolutionary analyses to a user-friendly, popular software that has been constantly updated from 
                                        DIVERGE to DIVERGE2 and to DIVERGE3 <a href="http://xungulab.website">(http://xungulab.website).</a>
                                    </p>
                                    <div style="background-color:#E3EEEA;margin-left:20px;margin-right: 20px;">
                                        <i>
                                            We have published over fifteen DIVERGE-related papers, and collectively the number of total citations is above 2000. Indeed, DIVERGE software series have become the mostly-cited bioinformatics tool among those that perform similar analyses.
                                        </i>
                                        </div>
                                        <hr>
                                    <h5>Transcriptome evolution analysis:</h5>
                                    <p>
                                        My achievements are as follows. 
                                        <i><u>(i) Statistical framework of transcriptome evolution</u></i>: for both microarray and RNA-seq, 
                                        it includes phylogenetic analysis of transcriptome evolution, expression clock 
                                        (constant evolutionary rate of expression divergence) and testing, ancestral transcriptome inference along a phylogeny, 
                                        and phylogenetic network analysis of multiple tissues/stages (e.g., see Gu 2004; Gu et al. 2005; Gu and Su 2007; Gu et al. 2013; Gu 2015a, Gu 2015b). 
                                        <i><u>(ii) Primate brain evolution</u></i> As one of first few groups, we (Gu and Gu 2003) discovered that gene expression in the human brain tends to 
                                        be upregulated since the split from chimpanzee. <i><u>(iii) Expression divergence between duplicates</u></i> Using the yeast as an example, 
                                        we (Gu et al. 2005) have demonstrated a rapid expression evolution after gene duplications, which has shown significant impacts (over 180 citations). 
                                        <i><u>(iv) Tissue-driven hypothesis</u></i>: Together with earlier microarray studies, Gu and Su (2007) proposed this hypothesis as an umbrella 
                                        for all tissue-specific effects on molecular and genomic evolution, For instance, the brain tissue 
                                        (or more generally, central nerve systems CNS) may have stronger functional constraints, the expressed genes may evolve slowly 
                                        in both coding sequence and expression. Recent RNA-Seq data have confirmed the major results. 
                                    </p>
                                    <div style="background-color:#E3EEEA;margin-left:20px;margin-right: 20px;">
                                        <i>
                                            As one of pioneer investigators for the emergence of this scientific paradigm, 
                                            I have established a model-based research program that help systematically analyze 
                                            the high throughput transcriptome data from microarray to RNA-seq.   
                                        </i>
                                        </div>
                                        <hr>
                                    <h5>Foundation of molecular evolution theory:</h5>
                                    <p style="margin-bottom: 10px;">
                                        In molecular evolution, the first thinking is population genetics 
                                        and the second thinking is phylogeny analysis of sequence, between which is 
                                        connected by the well-known Kimura’s formula underlying the neutral theory of 
                                        molecular evolution. However, explosive functional genomics calls for a third thinking, 
                                        tentatively termed ‘genotype-phenotype’ thinking. Though numerous empirical studies have 
                                        been reported, how one can integrate this ‘genotype-phenotype’ thinking into the theory of molecular 
                                        evolution remain a challenge. I have worked on this problem for ten years. 
                                        <i><u>(i) Genotype-phenotype model of molecular evolution</u></i>: I first proposed a statistical framework of molecular evolution 
                                        under the genotype-phenotype map (Gu 2007). Though the mapping model is mathematically abstract, 
                                        it successfully introduced the fundamental mapping parameter into the theory of molecular evolution, 
                                        that is, the rank (K) of genotype-phenotype map, which is the minimum between the rank of genotype (r) 
                                        and the rank of phenotype (n) (Gu 2014). <i><u>(ii) Effective estimation of gene pleiotropy:</u></i>
                                        I have developed a statistical method to estimate K=min (n, r) from a 
                                        multiple protein sequence alignment, which can be considered as an effective estimate of the gene pleiotropy (n), 
                                        or the number of fitness components related to the gene.  <i><u>(iii) Gene pleiotropy hypothesis of molecular evolution:</u></i> 
                                        After conducting large-scale data analyses, we proposed that a gene with high pleiotropy tends to evolve slowly. 
                                        <i><u>(iv) Molecular evolution under three thinking</u></i> recently, I (Gu 2015c) showed that, without positive selection, 
                                        sequence conservation (intensity of purifying selection) is determined by the effective population size, protein 
                                        structure stability, gene pleiotropy and expression effect.
                                    </p>
                                    <div style="background-color:#E3EEEA;margin-left:20px;margin-right: 20px;">
                                        <i>
                                            The foundation of molecular evolution stems from two scientific thinking:
                                             One is population genetics and the other is the ‘molecular clock’. The paradigm shift 
                                             of genome sciences demands an additional third thinking to connect between the model-driven evolutionary 
                                             theory and the data-driven comparative genomic exploration, that is, the genotype-phenotype thinking. I have studied this fundamental problem for a decade and believe that I am the only one in attempt to extend the classical
                                             theory of molecular evolution to integrate the third thinking. My work have been gradually recognized recently.   
                                        </i>
                                        </div>
                                        <hr>
                                    
                                        <h5>Patterns of genome evolution:</h5>
                                        <p style="margin-bottom: 10px;">
                                            While the pattern of molecular evolution such as molecular clock has been the central topic for half century, 
                                            this paradigm is having been shifted to the pattern of genome evolution. In the past two decades, I have made significant contributions on the following topics. 
                                            (i) We first discovered (Gu et al. 2002) that in mammalian genome, emergence of duplicate genes follows a three-component pattern: 
                                            Wave-I (recent duplications within mammals), Wave-II (genome duplications in early vertebrates), and the ancient component (during metazoan evolution or earlier). 
                                            (ii) We (Gu and Gu 2003) discovered that gene expression in the human brain tends to be upregulated since the split from chimpanzee Expression evolution in primate tissues. 
                                            (iii) We studied the role of gene duplication on genetic robustness, revealing that the contribution is significant but complicated. 
                                            (iv) We (Su et al. 2006) proposed the hypothesis that alternatively splicing isoforms can be fixed in one of duplicate genes, respectively. 
                                            (v) We developed the method of gene-content phylogeny to study the evolution (gain or loss) of gene content at the genome level. 
                                            (vi) Using TATA-box regulatory pathway as an example, we (Zou et al. 2012) demonstrated the expression divergence after gene duplication that may be environmentally-dependent: (vii) With the help of ENCODE data, 
                                            we (Zhou et al. 2013) studied the evolution of human transcription factor (TF) network. (viii) Evolution of phophalted site in metazoans. 
                                        </p>
                                        <div style="background-color:#E3EEEA;margin-left:20px;margin-right: 20px;">
                                            <i>
                                                With the growth of genomics for two decades, we have actively participated to the adventure of genomic explorations. 
                                                Though the genomic data changed rapidly, our research has always focused on the comparative genomic and evolution.    
                                            </i>
                                            </div>
                                            <hr>


                                            <h5>Statistical theory and methods for evolutionary genomics: A synthesis:</h5>
                                            <p style="margin-bottom: 10px;">
                                                In 2011, as single author, I published a monograph book ‘statistical theory and methods for evolutionary genomics’ by Oxford University Press, 258 pages. 
                                                This is the first book, and to our best knowledge is probably the only one at present in attempt to synthesize numerous literatures and my three-decade research in this field. 
                                                My goal is to develop a systematic framework from model presentations to application illustrations for both 
                                                research and education purposes. My life-time mission is to approach this goal and ultimately accomplish.
                                            </p>
                                </div>
                                <!-- End:: discription-tab  -->
                                <div class="tab-pane fade" id="pills-curriculum" aria-labelledby="pills-curriculum-tab" role="tabpanel">
                                    <h5>Papers with over hundred (100) citations</h5>
                                    <ul style="list-style-type:decimal;margin:20px">
                                        <li>Li WH, <strong>Gu X</strong> (1995) Statistical models for studying DNA sequence evolution. <strong><u><i>Physica A</i></u></strong>. 221:159-167. </li>
                                        <li><strong>Gu X</strong>, Fu YX, Li WH (1995) Maximum likelihood estimation of the heterogeneity of substitution rate among nucleotide sites. <strong><i><u>Molecular Biology and Evolution</u></i></strong> 12: 546-557.</li> 
                                        <li><strong>Gu X</strong>, Li WH (1995) The size distribution of insertions and deletions in human and rodent pseudogenes suggests the logarithmic gap penalty for sequence alignment. <strong><u><i>Journal of Molecular Evolution</i></u></strong> 40:464-473 </li>
                                        <li>Nei M, <strong>Gu X</strong>, Stinikova T (1997) Evolution by the birth-and-death process in multigene families of the vertebrate immune systems. <strong><i><u>PNAS</u></i></strong> 94:7799-7806. 
                                        </li>
                                        <li><strong>Gu X*</strong>, Zhang J (1997) A simple method for estimating the parameter of substitution rate variation among sites. <strong><u><i>Molecular Biology and Evolution</i></u></strong>. 14:1106-1113. </li>
                                        <li><strong>Gu X*</strong>(1998) Early metazoan divergence is about 830 million years ago. <strong><i><u>Journal of Molecular Evolution</u></i></strong> 47:369-371</li>
                                        <li><strong>Gu X*</strong> (1999) Statistical methods for testing functional divergence after gene duplication. <strong><u><i>Molecular Biology and Evolution</i></u></strong> 16:1664-1674. </li>
                                        <li><strong>Gu X*</strong> (2001) Maximum likelihood approach for gene family evolution under functional divergence. <strong><u><i>Molecular Biology and Evolution</i></u></strong>. 18: 453-464</li>
                                        <li>Wang Y, <strong>Gu X*</strong> (2001) Functional divergence in the caspase gene family and altered functional constraints: Statistical analysis and prediction. Genetics. 158:1311-1320. </li>	
                                        <li><strong>Gu, X*</strong>, Vander Velden K (2002) DIVERGE: Phylogeny-based Analysis for Functional-Structural Divergence of a Protein Family. <strong><u><i>Bioinformatics</i></u></strong> 18:500-501. </li>
                                        <li><strong>Gu, X*</strong>, Wang Y, Gu, J, Vander Velden K (2002) Evolutionary perspective for functional divergence of gene families and applications in functional genomics. <strong><u><i>Current Genomics</i></u></strong> . 3:201-212. </li>
                                        <li>Gaucher EA, <strong>Gu X</strong>,  Miyamoto, M, Benner, S. (2002) Predicting functional divergence in protein evolution by site-specific rate shifts. <strong><u><i>Trend in Biochemical Sciences (TiBS) </i></u></strong>27:315-321. </li>
                                        <li><strong>Gu, X*</strong>, Wang Y, Gu, J (2002) Age-distribution of human gene families showing equal roles of large and small-scale duplications in vertebrate evolution. <strong><u><i>Nature Genetics</i></u></strong> 31:205-209. </li>
                                        <li>Gu, Z, Steinmetz, LM, <strong>Gu, X</strong>, Scharfe, C., Davis RD, Li, WH (2003) Role of duplicate genes in genetic robustness against null mutations. <strong><u><i>Nature</i></u></strong> 421:63-66. </li>
                                        <li>Gu, J, <strong>Gu, X*</strong> (2003) Natural history and functional divergence of protein tyrosine kinases. <strong><u><i>Gene</i></u></strong> 317:49-57. </li>
                                        <li>Gu, J, <strong>Gu, X*</strong> (2003) Induced gene expression in human brain after the split from chimpanzee. <strong><u><i>Trends in Genetics</i></u></strong> 19:63-65. </li>
                                        <li><strong>Gu, X*</strong>. (2003) Genetic buffering, gene duplication, and evolution. <strong><u><i>Trends in Genetics</i></u></strong>. 19:354-356. </li>
                                        <li>Jiang, C., <strong>Gu X.</strong>, and T. Peterson (2004) Functional classification and prediction of Myb gene family in Arabidopsis and Rice. <strong><u><i>Genome Biology</i></u></strong> 5(7): R46. 
                                        </li>
                                        <li><strong>Gu, X*</strong>, Zhang Z, and Wei Huang (2005) Rapid evolution of expression and regulatory network after yeast gene/genome duplications. <strong><u><i>PNAS</i></u></strong> 102:707-712 
                                        </li>
                                        <li>Li, W-H, Yang, J and <strong>Gu, X</strong> (2005) Expression divergence between duplicates. <strong><u><i>Trends in Genetics</i></u></strong>. 21:602-607.  
                                        </li>
                                        <li>Lin H, Zhu W, Silva JC, <strong>Gu X</strong>, and Buell R (2006) Intron turnover of segmentally duplicated genes in rice. <strong><u><i>Genome Biology</i></u></strong> 7(5):R41.
                                        </li>
                                        <li><strong>Gu, X*</strong> (2006) A Simple Statistical Method for Estimating Type-II (Cluster-Specific) Functional Divergence of Protein Sequences. <strong><u><i>Molecular Biology and Evolution</i></u></strong> 23:1937-1945.  
                                        </li>
                                        <li>Su Z, Wang J, Yu J, Huang X and <strong>Gu, X*</strong> (2006) Evolution of Alternative Splicing after Gene Duplications. <strong><u><i>Genome Research</i></u></strong> 16:182-189. 
                                        </li>
                                        <li><strong>Gu, X*</strong>, Zou Y, Su Z, Huang W, Zhou Z, Arendsee Z, Zeng Y (2013) An Update of DIVERGE Software for Functional Divergence Analysis of Protein Family. <strong><u><i>Molecular Biology and Evolution</i></u></strong> 30(7) 1713-1719. 
                                        </li>

                                    </ul>

                                    <h5>Papers with citations between 50 and 99</h5>
                                    <ul style="list-style-type:decimal;margin:20px">
                                        <li><strong>Gu X</strong> , Li WH (1992) Higher rates of amino acid substitution in rodents than in humans.  <strong><u><i>Molecular Phylogenetics and Evolution</i></u></strong>  1:211-214</li>
                                        

                                        <li>Huang W, Chang B HJ, Gu X, Hewett-Emmett D, Li WH (1997) Sex differences in mutation rate in higher primates estimated from AMG intron sequences. Journal of Molecular Evolution. 44:463-465. 
                                        </li>
                                        <li>Gu X, Li WH (1998) Evolutionary distances under stationary and nonstationary models of nucleotide substitutions. PNAS 95: 5899-5905. 
                                        </li>
                                        <li>Zhang J, Gu X (1998) Correlation between substitution rate and rate variation among sites. Genetics 149:1615-1625. 
                                        </li>
                                        <li>Gu X, Hewett-Emmett D, Li WH (1998) Direct mutational pressure affects the amino acid composition and hydrophobicity of proteins in bacteria. Genetica 102/103; 383-391 
                                        </li>
                                        <li>Gu X*, Nei M (1999) Locus-specificity of polymorphic alleles and evolution by birth-and-death process in mammalian MHC genes. Molecular Biology and Evolution 16:147-156. 
                                        </li>
                                        <li>Su C, Jakobsen I, Gu X, Nei M (1999) Diversity and evolution of T-cell receptor variable region genes in mammals and birds. Immunogenetics 50:301-308. 
                                        </li>
                                        <li>Wang Y, Gu X* (2000) Evolutionary patterns of gene families generated in the early stage of vertebrates. Journal of Molecular Evolution. 51:88-96. 
                                        </li>
                                        <li>Gu, X* (2003) Functional divergence in protein (family) sequence evolution. Genetica 118:133-141. 
                                        </li>
                                        <li>Jiang, C., Gu, J., Gu, X., and T. Peterson (2004) Ordered origin of the typical two- and three-repeat Myb genes.  Gene 326:13-22. 
                                        </li>
                                        <li>Gu, X* (2004) Statistical framework for phylogenetic analysis of expression profiles. Genetics 167: 531-542.  
                                        </li>
                                        <li>Gu, X*, Zhang H (2004) Genome phylogeny inference based on gene contents. Molecular Biology and Evolution  21:1401-1408.  
                                        </li>
                                        <li>Zhang, Z, Gu, J and Gu, X* (2004) How much expression divergence after yeast gene duplication could be explained by regulatory motif evolution? Trends in Genetics 20:403-407. 
                                        </li>
                                        <li>Leebens-Mack, J, Vision T, Brenner E,…Gu X … (2006) Taking the first steps towards a standard for reporting on phylogenies: Minimal Information About a Phylogenetic Analysis (MIAPA). OMIC Integrated Biology. 10:231-237. 
                                        </li>
                                        <li>Gu, X*, and Su, Z (2007) Tissue-driven hypothesis of genomic evolution and sequence-expression correlations. PNAS 104:2279-2784.  
                                        </li>
                                        <li>H Lin, S Ouyang, A Egan, K Nobuta, B Haas, W Zhu, X Gu, JC Silva, BC Meyers, C R Buell (2008) Characterization of paralogous protein families in rice BMC Plant Biology 8:18.  
                                        </li>
                                        <li>Haining Lin, Gaurav Moghe, Shu Ouyang, Amy Iezzoni, Shin-Han Shiu, Xun Gu*, C. Robin Buell* (2010) Comparative analyses reveal distinct sets of lineage-specific genes within Arabidopsis thaliana. BMC Evolutionary Biology. 10:41 
                                        </li>
                                        



                                    </ul>

                                </div>
                                
                            </div>














                            
                            
                        </div>

                    </div>
                    <!--/ article -->
                </div>

            </div>
        </div>
        <!-- Container /- -->
    </div>

    <!-- Start:Footer Section
==================================================-->
    <footer class="footertwo-section">
        <div class="container">
            <div class="row">

                <!-- Start: logo -->
                <div class="col-lg-3 col-md-6 col-sm-12">
                    <div class="footer_logo">
                        <h4>Computational Genomics and Evolution Lab</h4>
                        <ul>
                            <li>
                                <a href="">xgu@iastate.edu </a>
                            </li>
                            <li>
                                <a href="javascript:;">515-294-8075</a>
                            </li>
                            <li>
                                <a href="javascript:;">332 Science II Hall, Departments of Zoology/Genetics<br>Iowa State University<br>Ames, IA, 50011, USA</a>
                            </li>
                        </ul>
                        
                    </div>
                </div>
                <!-- End: logo -->

                <!-- Start: Quick Link -->
                <div class="col-lg-3 col-md-6 col-sm-12">
                    <div class="foo_widget quick_lnk">
                        
                    </div>
                </div>
                <div class="col-lg-3 col-md-6 col-sm-12">
                    <div class="foo_widget quick_lnk">
                        
                    </div>
                </div>
                <!-- End: Quick Link -->

                <!-- Start: Useful Link -->
                <div class="col-lg-3 col-md-6 col-sm-12">
                    <div class="foo_widget useful_lnk">
                        <h5>Useful Links</h5>
                        <ul>
                            <li>
                                <a href="http://www.sciencemag.org/">Science</a>
                            </li>
                            <li>
                                <a href="http://www.nature.com/">Nature</a>
                            </li>
                            <li>
                                <a href="https://genome.cshlp.org/">Genome Research</a>
                            </li>
                            <li>
                                <a href="http://www.ncbi.nlm.nih.gov/">National Center for Biotechnology Information</a>
                            </li>
                            <li>
                                <a href="https://www.ebi.ac.uk/">Pôle Bio-Informatique Lyonnais</a>
                            </li>
                        </ul>
                    </div>
                </div>
                <!-- End: Social Link-->

                
            </div>
        </div>
        <!-- Start:Subfooter -->
        <div class="subfooter">
            <p>Copyright &copy; 2021.Gu's Lab All rights reserved.</p>
        </div>
        <!-- End:Subfooter -->
    </footer>
    <!-- End:Footer Section
========================================-->


    <!-- Scripts
========================================-->
    <!-- jquery -->
    <script src="static/js/jquery-1.12.4.min.js"></script>
    <!-- plugins -->
    <script src="static/js/plugins.js"></script>
    <!-- Bootstrap -->
    <script src="static/js/bootstrap.min.js"></script>
    <script src="static/js/slick.min.js"></script>
    <script src="static/js/jquery.stellar.min.js"></script>


    <!-- Custom Scripts
========================================-->
    <script src="static/js/main.js"></script>

</body>

</html>